courses/fundamentals_of_ada/140_access_types.rst

```diff
@@ -454,7 +458,10 @@ Introduction to Accessibility Checks (1/2)
                -- Nested subprogram, enclosing + 1, here 2
                O2 : aliased Integer;
 
-* Access **types** can only access objects that are at **same or lower depth**
+* Objects can be access by access **types** that are at **same depth or deeper**
```

Comment:
should be "accessed by", but better would be "referenced by" or "pointed to by"

Reply:
agreed